tests/functional/commands/validate/lib/models/basic_application.yaml

```diff
@@ -0,0 +1,41 @@
+Conditions:
+  TestCondition:
+    Fn::Equals:
+      - True
+      - False
+Resources:
+  BasicApplication:
+    Type: 'AWS::Serverless::Application'
+    Properties:
+      Location:
+        ApplicationId: arn:aws:serverlessrepo:us-east-1:123456789012:applications/hello-world
+        SemanticVersion: 1.0.2
+
+  NormalApplication:
+    Type: 'AWS::Serverless::Application'
```

Comment:
Files with `AWS::Serverless::Application` types need to be removed. The `AWS::Serverless::Applicaiton` Resources require credentials in order for the Transform to work. For now, we have excluded these in the validation.

Reply:
Yes ... I have removed them manually in the next commit. I have to update the script to remove these files automatically from next time